src/controllers/wallet/atomic-swap/tx-proposal.controller.js

```diff
@@ -15,10 +15,7 @@ const {
   constants: { HATHOR_TOKEN_CONFIG, TOKEN_MINT_MASK, TOKEN_MELT_MASK },
   wallet: oldWallet,
 } = require('@hathor/wallet-lib');
-const {
-  assembleTransaction,
-  serviceCreate
-} = require('../../../services/atomic-swap.service');
+const atomicSwapService = require('../../../services/atomic-swap.service');
```

Comment:
Removed the object destructuring to allow this service's methods to be mocked on the tests.